kairon/api/app/routers/bot/bot.py

```
@@ -1700,3 +1701,92 @@
+    List all documents within a collection.
+    """
+    records = DataProcessor.list_collection_data(
+        bot=current_user.get_bot(),
+        name=payload["collection_name"]
+    )
+    return Response(data=records)
+
+
+@router.post("/collections/data/add", response_model=Response)
+async def add_collection_data(
+    payload: dict = Body(...),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)